<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>日期时间</string>
     </property>
     <widget class="QPushButton" name="pushButton_readtime">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>读取当前日期时间</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>30</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>字符串显示</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>69</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>时间</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_time">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>H:mm:ss</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_time">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>60</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>99:99:99;_</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_qdebugtime">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>qDebug</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_settime">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>90</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>设置时间</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>69</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>日期</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_data">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_date">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>150</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>9999:99:99</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>411</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_qDeugdate">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>180</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>qDebug</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_setdate">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>180</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>设置日期</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>411</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>69</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>日期时间</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit_dataTime">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>240</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>yyyy/M/d H:mm:ss</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_timedate">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>240</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>9999-99-99 99:99:99</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_qDebugtimedate">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>270</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>qDebug</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_settimedate">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>270</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>设置时间日期</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>日历选择</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>69</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>选择日期：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>296</width>
        <height>229</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
